electron_app/src/electron-main.js

```diff
@@ -322,14 +323,14 @@ app.on('ready', () => {
         webPreferences: {
             preload: preloadScript,
             nodeIntegration: false,
-            sandbox: true,
-            enableRemoteModule: false,
             // We don't use this: it's useful for the preload script to
             // share a context with the main page so we can give select
             // objects to the main page. The sandbox option isolates the
             // main page from the background script.
+            sandbox: true,
```

Comment:
Could you move this back to where it was to make the diff cleaner?

Reply:
ok